### PR TITLE
Added some more clarity around config naming

### PR DESCRIPTION
**Pull request by ekfuhrmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,10 @@ Include the CDN hosted script and configure the `<parts-kit>` custom element wit
 In the future, we'll have CMS plugins that automatically setup and configure of your parts kit.
 
 ```html
-<script type="module" src="https://unpkg.com/@viget/parts-kit@^0/lib/parts-kit.js"></script>
+<script
+  type="module"
+  src="https://unpkg.com/@viget/parts-kit@^0/lib/parts-kit.js"
+></script>
 <parts-kit config-url="YOUR-URL-HERE"></parts-kit>
```
Comment (andrewmosby) [inline README.md:38]: This is the part I was confused on. E.g. that it needs to be `/parts-kit/config.json` or `/parts-kit.json`. I tried lots of patterns to get the URL to where the file was located, but didn't realize I needed to include the `config.json` bit.

Comment (jeremyfrank) [reply README.md:38]: For clarity, I'd suggest the following:

```suggestion
<parts-kit config-url="/path/to/config.json"></parts-kit>
```

Comment (andrewmosby) [reply README.md:38]: Yes, exactly!

Comment (ekfuhrmann) [reply README.md]: Good suggestion!

**Review by joshuapease — APPROVED** [review]
These are all great tweaks. Thanks everyone!

**Review by andrewmosby — APPROVED** [review]

**Pull request merged**